src/index.js

```diff
@@ -0,0 +1,50 @@
+import CarModel from './model/CarModel.js';
+import InputView from './view/InputView.js';
+
+function printOneGame(nameList, cars) {
+  for (let i = 0; i < nameList.length; i++) {
+    const carOutput = '-'.repeat(cars[i].position);
+    console.log(`${nameList[i]} : ${carOutput}`);
+  }
+  console.log('');
+}
+
+// 입출력 예시
+export async function run() {
+  const cars = [];
+
+  const nameList = await InputView.getNameList();
+  const count = await InputView.getCount();
+
+  nameList.forEach(name => {
+    const car = new CarModel(name);
+    cars.push(car);
+  });
```

Comment:
다음과 같이 간소화 할 수 있습니다.

```suggestion
  const nameList = await InputView.getNameList();
  const count = await InputView.getCount();

  const cars = nameList.map(name => new CarModel(name));
```

Reply:
cars가 앞에있고 뒤에 처리하는 로직이 있으니 훨씬더 직관적으로 보여서 좋은것 같습니다. 감사합니다...!